go.mod

```diff
@@ -1,8 +1,9 @@
 module github.com/bitcoin-sv/go-paymail
 
-go 1.21.5
+go 1.22.4
 
 require (
+	github.com/bitcoin-sv/spv-wallet/models v1.0.0-beta.14
```

Comment:
As I recall our conversation, we wanted not to make `go-paymail` dependant to `spv-wallet/models`.

Maybe in this repo we could introduce the same `SPVError`, like `PaymailError` and in `spv-wallet/engine/spverrors/getError` handle also this new struct 🤔 ?